plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/extension/FileOpenSaveExtensionPoint.java

```diff
@@ -83,11 +83,7 @@
       fileDialogOperation.setRepositoryObject( repositoryObject );
       fileDialogOperation.setProvider( repositoryOpenSaveDialog.getProvider() );
     } else {
-      if ( fileDialogOperation.getCommand().equals( FileDialogOperation.SAVE ) ) {
-        fileDialogOperation.setPath( repositoryOpenSaveDialog.getParentPath() );
-      } else {
-        fileDialogOperation.setPath( repositoryOpenSaveDialog.getPath() );
-      }
+      fileDialogOperation.setPath( repositoryOpenSaveDialog.getPath() );
```

Comment:
By changing this to use Path instead of Parent did this impact spoon.java's use of the Extension point?

Reply:
Not sure; will investigate more.